src/workloads/execution/MultiPlanningReadsALotOfData.yml

```
@@ -0,0 +1,98 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+ Create collection with multiple indexes and run queries for which multi planner needs to read a lot
+ of data in order to pick the best plan..
+
+Keywords:
+- indexes
+
+Actors:
+- Name: InitCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: &MaxPhase 2
+      PhaseConfig:
+        Repeat: 1
+        Database: &Database test
+        Operations:
+        - OperationName: RunCommand
+          OperationCommand:
+            createIndexes: &Collection Collection0
+            indexes:
+            - key:
+                flag_a: 1
+                flag_b: 1
+              name: flags
+            - key:
+                int_a: 1
+              name: int
+
+- Name: DisablePlanCache
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhase
+      PhaseConfig:
+        Repeat: 1
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            internalQueryDisablePlanCache: true
+
+- Name: Insert
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhase
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Threads: 1
+        CollectionCount: 1
+        DocumentCount: 2e6
+        BatchSize: 1000
+        Document:
+          flag_a: {^RandomInt: {min: 0, max: 1}}
```

[nit/ optional] We could anchor this as `&randomInt` and reuse `*randomInt`:
```suggestion
          flag_a: &randomInt {^RandomInt: {min: 0, max: 1}}
```

---

Added